src/main.cpp

```diff
@@ -258,6 +247,18 @@ int main(void) {
     if (irq == 0)
         __disable_irq();
 
+    buddy::hw::BufferedSerial::uart2.Open();
```

Comment:
this can be a problem in the future, the uart2 probably uses something from RTOS which is not inited yet ... therefore the move after eeprom init